### PR TITLE
Labs Transcripts: google_drive migration

### LINKED ISSUE #6399: [OAuth] Labs/Transcript migrate google_drive

### PR DESCRIPTION
**Pull request by spolu**
## Description

Fixes https://github.com/dust-tt/dust/issues/6399

- move setup to new `oauth`
- support both nango and `oauth`
- migration script (will also be used for `gong`)

## Risk

Low. Labs

## Deploy Plan

- deploy `front`

### CODE REVIEW

File: front/pages/w/[wId]/assistant/labs/transcripts/index.tsx [hunk 1]
```diff
@@ -286,33 +279,30 @@ export default function LabsTranscriptsIndex({
...
-        `labs-transcripts-workspace-${owner.id}-user-${user.id}`,
-        transcriptsConfigurationState.provider
-      );
-      const {
-        connectionId: newConnectionId,
-      }: { providerConfigKey: string; connectionId: string } = await nango.auth(
-        nangoDriveConnectorId,
-        nangoConnectionId
-      );
+    if (transcriptsConfigurationState.provider !== "google_drive") {
```
Comment (flvndvd) [inline front/pages/w/[wId]/assistant/labs/transcripts/index.tsx:282]: IIRC, the `try/catch` was used to catch errors coming from `saveOauthConnection` that does a `fetch` call, can we either keep it or move it to the function?

**Review by flvndvd — APPROVED** [review]

File: front/pages/w/[wId]/assistant/labs/transcripts/index.tsx (hunk 1 above)
Comment (spolu) [reply front/pages/w/[wId]/assistant/labs/transcripts/index.tsx:293]: Added a try catch

**Comment by spolu** [comment]
Setup works, didn't test processing end to end but will move forward as is :+1:

**Comment by spolu** [comment]
@albandum can you test an end to end setup on your end once this is live :pray: 
Will give it a try as well.

**Pull request merged**